docs/introduction/performance/scenarios.rst

```diff
@@ -133,123 +148,138 @@ Een afgeronde enkele zaak opvragen, met een resultaat, een besluit, *2 zaakobjec
 
 **Documenten API**
 
-* 1x ENKELVOUDIGINFORMATIEOBJECT aanmaken (`POST /api/v1/enkelvoudiginformatieobjecten`)
+* 1x ENKELVOUDIGINFORMATIEOBJECT aanmaken (``POST /api/v1/enkelvoudiginformatieobjecten``)
 
-#### Besluit toevoegen (11)
+Besluit toevoegen (11)
+~~~~~~~~~~~~~~~~~~~~~~
 
 **Besluiten API**
 
-* 1x BESLUIT aanmaken (`POST /api/v1/besluiten`)
+* 1x BESLUIT aanmaken (``POST /api/v1/besluiten``)
 
-#### Resultaat toevoegen (12)
+Resultaat toevoegen (12)
+~~~~~~~~~~~~~~~~~~~~~~~~
 
 **Zaken API**
 
-* 1x RESULTAAT aanmaken (`POST /api/v1/resultaten`)
+* 1x RESULTAAT aanmaken (``POST /api/v1/resultaten``)
 
-### Test specificatie
+Test specificatie
+-----------------
 
-#### Gebruik van scenario's
+Gebruik van scenario's
+~~~~~~~~~~~~~~~~~~~~~~
 
 Niet elk scenario wordt even vaak uitgevoerd. Een zaak wordt bijvoorbeeld vaker opgevraagd dan aangemaakt. In de onderstaande tabel wordt bijvoorbeeld voor elke 20x "Zaken overzicht opvragen", 10x "Zaak aanmaken" uitgevoerd. Vervolgens is dit omgezet naar een percentage, er van uitgaande dat alle scenario's 100% vertegenwoordigt.
 
 Om de praktijk verder te benaderen wordt voor elk scenario een bepaalde wachttijd genomen. De wachttijd is de tijd die een echte gebruiker bijvoorbeeld nodig heeft om gegevens in te vullen in de gebruikersinterface. Deze wachttijd vertaald zich naar de tijd tussen scenario's. In de onderstaande tabel wordt bijvoorbeeld bij het uitvoeren van "Zaak aanmaken" eerst tussen 0 en 10 minuten gewacht (gemiddeld 5 minuten).
 
 De wachttijd staat voor de snelheid waarmee gebruikers bepaalde acties in het virtuele systeem uitvoeren en daarmee de belasting die ze veroorzaken.
 
-| # | Scenario | Verdeling | Verdeling % | Gemiddelde wachttijd (minuten) | Wachttijd spreiding (minuten) |
-|---|---|---|---|---|---|
-| 1 | Zaken overzicht opvragen | 20 | 20% | 2 | 0 - 4 |
-| 2 | Zaken overzicht opvragen voor specifiek zaaktype | 10 | 10% | 2 | 0 - 4 |
-| 3 | Zaken zoeken op locatie | 1| 1% | 1 | 0 - 2 |
-| 4 | Zaken zoeken op persoon | 10 | 10% | 1 | 0 - 2 |
-| 5 | Zaak details opvragen | 8 | 8% | 2 | 0 - 4 |
-| 6 | Geschiedenis opvragen | 2 | 2% | 3 | 0 - 6 |
-| 7 | Zaak aanmaken | 10 | 10% | 5 | 0 - 10 |
-| 8 | Status toevoegen | 20 | 20% | 2 | 0 - 4 |
-| 9 | Betrokkene toevoegen | 3 | 3% | 3 | 0 - 6 |
-| 10 | Document toevoegen | 12 | 12% | 4 | 0 - 8 |
-| 11 | Besluit toevoegen | 2 | 2% | 3 | 0 - 6 |
-| 12 | Resultaat toevoegen | 2 | 2% | 3 | 0 - 6 |
-| | **Totaal** | 100 | 100% |  |  |
-
-#### Virtuele gebruikers
+=== ==============================  ======  ======  ======  ======
+#   Scenario                        Verdeling       Wachttijd
+--- ------------------------------  --------------  --------------
+.   .                               Abs.    %       Gem.    Spr.
+=== ==============================  ======  ======  ======  ======
+1   Zaken overzicht opvragen        20      20%     2       0 - 4
```

Comment:
The performance tests now have the same waiting time for all tasks - 0-5 minutes

Reply:
True, I'll change this.